Source/Microsoft.Teams.Apps.CompanyCommunicator/Controllers/SentNotificationsController.cs

```diff
@@ -81,6 +82,9 @@ public async Task<IEnumerable<SentNotificationSummary>> GetSentNotificationsAsyn
                     Throttled = notificationEntity.Throttled,
                     TotalMessageCount = notificationEntity.TotalMessageCount,
                     IsCompleted = notificationEntity.IsCompleted,
+                    SendingStartedDateTime = notificationEntity.SendingStartedDateTime.HasValue
```

Comment:
Let's have this just be notificationEntity.SendingStartedDate and not have the value check. If it is null, the front end can/should handle it - MinValue may confuse it.

Reply:
Addressed.